# 2015.02.05 17:18:08 IST
from onep.core.util.Enum import enum
OnepStatus = enum('ONEP_OK', 'ONEP_FAIL', 'ONEP_ERR_INCOMPATIBLE_VERSION', 'ONEP_ERR_MEM_FAIL', 'ONEP_ERR_BAD_REQUEST', 'ONEP_ERR_BAD_ARGUMENT', 'ONEP_ERR_SYSTEM', 'ONEP_ERR_CONNECT_FAIL', 'ONEP_ERR_AUTHEN_FAIL', 'ONEP_ERR_EVENT_CHANNEL', 'ONEP_ERR_NO_DATA', 'ONEP_ERR_DUPLICATE', 'ONEP_ERR_NOT_SUPPORTED', 'ONEP_ERR_INCONSISTENCY', 'ONEP_ERR_IPC_FAILED', 'ONEP_ERR_NOT_CONNECTED', 'ONEP_ERR_AAA_NO_SERVICE_PROVIDER', 'ONEP_ERR_AAA_SERVER_UNREACHABLE', 'ONEP_ERR_AAA_NO_AUTHOR_DATA', 'ONEP_ERR_SERVICE_DISABLED', 'ONEP_ERR_SERVICE_UNAVAILABLE', 'ONEP_ERR_SERVICE_INVALID', 'ONEP_ERR_RESOURCE_BUSY', 'ONEP_ERR_SESSION_MAX', 'ONEP_ERR_START_ONEP', 'ONEP_ERR_VTY_ALREADY_OPENED', 'ONEP_ERR_VTY_ALREADY_CLOSED', 'ONEP_ERR_API_NOT_IMPLEMENTED', 'ONEP_PARTIAL_SUCCESS', 'ONEP_ERR_NO_SPACE', 'ONEP_EOK_EVENTQ_FULL', 'ONEP_ERR_EAGAIN', 'ONEP_ERR_NOT_ACCESSIBLE', 'ONEP_ERR_UNRESOLVED_VALUE', 'ONEP_ERR_ROLLBACK_FAILED', 'ONEP_ERR_CLEANUP_FAILED', 'ONEP_ERR_EXCEEDS_RANGE', 'ONEP_WARN_CLEAR_TEXT_TRANSPORT', 'ONEP_ERR_INVALID_VRF', 'ONEP_ERR_NOT_READY', 'ONEP_ERR_EXCEEDS_MAX_ALLOWANCE', 'ONEP_ERR_TIMEOUT', 'ONEP_IF_NOT_SUPPORTED')

# decompiled 1 files: 1 okay, 0 failed, 0 verify failed
# 2015.02.05 17:18:08 IST
